### ISSUE TITLE
Weight C in TransH missing

### ISSUE THREAD

**Issue by Norual** [thunlp/OpenKE#95]
Hello,

I have read the paper about TransH. They have mentioned a weight C as one of the hyperparameter. Is it included in the code? If yes can you please tell me where to set it? If it is not included would you explain why?

Thanks.

**Comment by THUCSTHanxu13** [comment]
I have added normalization operations for TransX models. And the current implementation can achieve the state-of-the-art results on FB15K237 and WN18RR. The hyper-parameters and script code can be founded in the folder "./examples/".

**Comment by THUCSTHanxu13** [comment]
We have tried various methods to add normalization operations for TransX models. And finally, the current normalization implementation can achieve the best performance on FB15K237 and WN18RR. Maybe you can try other constraints. If you achieve better results, please pull requests for us : )
